cmd/internal/add-license-header/add-license-header.go

```diff
@@ -81,3 +89,9 @@ func main() {
 		fmt.Println("Wrote license header to " + file)
 	}
 }
+
+func main() {
+	projectRoot := getProjectRoot()
+	files := collectFiles(projectRoot)
+	writeLicenseHeader(projectRoot, files)
```

Comment:
btw, if the function definitions are only to trim down the body size of the one `main` function; there isn't too much point. Would we ever use `getProjectRoot` or `collectFiles` or `writeLicenseHeader` anywhere else?